### PR TITLE
impliment function_make-set

### PR DESCRIPTION
**Pull request by biolee**
```go
// See https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_make-set
func (b *builtinMakeSetSig) eval(row []types.Datum) (d string, err error) {
	var bitMask int
	var s []string

	switch maskValue := row[0].(type) {
	case int:
		bitMask = maskValue
	default:
		fmt.Println("row[0] must be a 'int'")
		return "", errors.New("row[0] must be a 'int'")
	}

	for index, value := range row[1:] {
		switch valueValue := value.(type) {
		case string:
			if (value != "") && (index&bitMask != 0) {
				s = append(s, valueValue)
			}
		}
	}
	
	d = strings.Join(s, ",")
	return
}
```

### CODE REVIEW

**Comment by shenli** [comment]
@biolee Thanks for your PR!
Please add some unit test.

File: expression/builtin_string.go [hunk 1]
```diff
@@ -1394,8 +1394,29 @@ type builtinMakeSetSig struct {
 }
 
 // See https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_make-set
-func (b *builtinMakeSetSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("make_set")
+func (b *builtinMakeSetSig) eval(row []types.Datum) (d string, err error) {
+	var bitMask int
+	var s []string
+
+	switch maskValue := row[0].(type) {
```
Comment (shenli) [inline expression/builtin_string.go]: Use row[0].Kind() to check the type.

File: expression/builtin_string.go [hunk 2]
```diff
@@ -1394,8 +1394,29 @@ type builtinMakeSetSig struct {
...
+	case int:
+		bitMask = maskValue
+	default:
+		fmt.Println("row[0] must be a 'int'")
```
Comment (shenli) [inline expression/builtin_string.go]: Do not use fmt to print log.

File: expression/builtin_string.go [hunk 3]
```diff
@@ -1394,8 +1394,29 @@ type builtinMakeSetSig struct {
...
+		return "", errors.New("row[0] must be a 'int'")
+	}
+
+	for index, value := range row[1:] {
+		switch valueValue := value.(type) {
```
Comment (shenli) [inline expression/builtin_string.go]: Use Datum.ToString() to convert data to string.

File: expression/builtin_string.go (hunk 1 above)
Comment (coocood) [inline expression/builtin_string.go]: The row is not the arguments.

File: expression/builtin_string.go [hunk 4]
```diff
@@ -1394,8 +1394,29 @@ type builtinMakeSetSig struct {
 }
 
 // See https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_make-set
-func (b *builtinMakeSetSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("make_set")
+func (b *builtinMakeSetSig) eval(row []types.Datum) (d string, err error) {
```
Comment (coocood) [inline expression/builtin_string.go]: The returned value type may not be changed.

**Comment by coocood** [comment]
@biolee 
I'sorry, this function has been implemented by another contributor.

**Pull request closed without merging**